### PR TITLE
 Remove term status field definition in order to use core

### PR DESCRIPTION
**Pull request by chrfritsch**
Make sure these boxes are checked before submitting your pull request - thank you!

- [ ] All coding styles are fulfilled. ([How to check for cs issues?](https://github.com/BurdaMagazinOrg/thunder-dev-tools/blob/master/README.md#code-style-guidelines))
- [ ] All tests are running locally. ([How to run the test?](https://github.com/BurdaMagazinOrg/thunder-distribution/blob/develop/docs/development.md#how-to-run-the-tests))
- [ ] Necessary update hooks are provided.
- [ ] User roles have correct access for new introduced permission.
- [ ] Every thunder module has a README.md in its root. Follow [this guidelines](https://www.drupal.org/node/2181737), but we don't need every topic.
- [ ] Code is covered with well-balanced amount of inline comments.

If you are really awesome, then your feature is covered by additional tests. Well done!

### CODE REVIEW

File: composer.json
Comment (dbosen) [inline composer.json]: Same as with #542, lets wait until this is merged into core, till we merge  this PR

**Review by dbosen — CHANGES_REQUESTED** [review]

**Comment by chrfritsch** [comment]
Closing this in favor of #582. We have all the code changes there plus an upgrade path.

**Pull request closed without merging**
